docs/SPEC.md

```diff
@@ -1172,7 +1145,7 @@ All such values must have a corresponding builtin statement to declare the exist
     MonoType     = Tvar | BasicType | ArrayType | RecordType | FunctionType .
     Tvar         = "A" … "Z" .
     BasicType    = "int" | "uint" | "float" | "string" | "bool" | "time" | "duration" | "bytes" | "regexp" .
-    ArrayType    = "[" MonoType "]" .
+    ArrayType    = [ "stream | "vector" ] "[" MonoType "]" .
```

Comment:
I'd move the stream and vector to their own definition

```
StreamType = "stream" "[" MonoType "]"
// etc
```

or at least rename `ArrayType` to `CollectionType`